drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/downloading/source/PhantomJSGitHubBitbucketSource.java

```diff
@@ -20,32 +18,44 @@
 
     private static String BASE_DOWNLOAD_URL = "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-";
 
+    private static String lastPhantomJSRelease = "2.1.1";
+
     public PhantomJSGitHubBitbucketSource(HttpClient httpClient, GitHubLastUpdateCache gitHubLastUpdateCache) {
```

Comment:
Currently, there is: 
~~~ java
* A slightly changed {@link GitHubSource} implementation handling PhantomJS binaries. The latest version is retrieved
* from list of GH tags and download URL is constructed to use Bitbucket downloads storage.
~~~
so I thought that it is enough